buildtest/cli/__init__.py

```diff
@@ -12,6 +13,11 @@
 from buildtest.defaults import console
 from buildtest.schemas.defaults import schema_table
 
+# Variables needed to show all sub commands and their help mesaage
+help1 = "-H" in sys.argv
+help2 = "--help-all" in sys.argv
+show_all_help = help1 or help2
```

Comment:
this seems silly we dont need two variables `help1` and `help2`. It also not readable.

This can be a one liner

```python
show_all_help = "-H" in sys.argv or "--help-all" in sys.argv
```

Reply:
Ill make sure to fix that!